lib/capybara/node/actions.rb

```diff
@@ -258,13 +260,16 @@ def unselect(value = nil, from: nil, **options)
       #   @param [String] locator                  Which field to attach the file to
       #   @param [String, Array<String>] paths     The path(s) of the file(s) that will be attached
       #
-      #   @option options [Symbol] match (Capybara.match)     The matching strategy to use (:one, :first, :prefer_exact, :smart).
```

Comment:
[note] `match` possible values are described in `Capybara.configure`, so I omit them here.

https://github.com/teamcapybara/capybara/blob/0f5fb07d9b7954ae3bd0d0924dcfb150d8e8ee4c/lib/capybara.rb#L92

Reply:
Please leave the options in the description.  It's easier to use the documentation that way.

Reply:
I'm worried that describing both is not DRY, which will cause a mismatch when a new option will be added. 🤔 

But your opinion about the usability also makes sense, so I'll follow you finally.

Reply:
I reverted `(:one, :first, :prefer_exact, :smart)`. Please review again.